core/lerna/schemas/lerna-schema.json

```diff
@@ -1487,6 +1490,10 @@
       "continueIfNoMatch": {
         "type": "boolean",
         "description": "Don't fail if no package is matched."
+      },
+      "summaryFile": {
+        "type": "string",
+        "description": "A json summary report will be generated after all packages have been successfully published."
```

Comment:
I should have caught this before but please expand on this description (in its two locations in this PR) so that it's clear you can pass a directory path to it

Reply:
Good call, I've updated